management_app/templates/courses/offerings.html

```diff
@@ -0,0 +1,176 @@
+{% extends 'base.html' %}
+
+{% block title %}Courses{% endblock %}
+{% block header %}Courses{% endblock %}
+
+<!-- {% block head %}
+  {{ super() }}
+  <link rel="stylesheet" href="{{ url_for('static', filename='courses.css') }}">
+{% endblock %} -->
+
```

Comment:
Suggestion but minor: if you don't need code in comments, you can clean them, so it could be more readable.